packages/server-wallet/src/__test-with-peers__/create-and-close-channel/direct-funding.test.ts

```diff
@@ -87,6 +87,19 @@ it('Create a directly funded channel between two engines ', async () => {
     status: 'running',
     turnNum: 3,
   });
+
+  // TODO (DirectFunder) Remove hacky expectation
+  (['a', 'b'] as const).map(key => {
+    const objective = (peerEngines[key] as any).richObjectives[channelId];
+    const signatures = [{signer: expect.any(String)}, {signer: expect.any(String)}];
+    expect(objective).toMatchObject({
```

Comment:
For this expectation to pass, I needed to either:
- change `updateChannelFundingFor` to use `holdingUpdated`
- change the implementation of `updateChannelFunding` to match the implementation of `holdingUpdated`

I opted for the former, as explained in #3438.